wrappers/azurekeyvault/azurekeyvault.go

```diff
@@ -365,3 +387,16 @@ func ParseKeyVersion(kid string) string {
 	keyVersionParts := strings.Split(kid, "/")
 	return keyVersionParts[len(keyVersionParts)-1]
 }
+
+func cloudConfigAndKeyVaultDNSSuffixFromName(name string) (cloud.Configuration, string, error) {
+	switch name {
```

Comment:
Should this be `switch strings.ToUpper(name) {`? The environment values are typically provided in pascal case like AzurePublicCloud, AzureChinaCloud, ...